izumi-reflect/izumi-reflect/src/test/scala/izumi/reflect/test/CurrentDottySupportExtentTest.scala

```diff
@@ -18,60 +18,52 @@ trait CurrentDottySupportExtentTest extends TagAssertions {
   trait Invariantoid[V]
   trait SubInvariantoid[V] extends Invariantoid[V] with Traitoid
 
-  // FIXME: report upstream: LTT macro calls do not work inside class body / inside normal test
```

Comment:
Odd that it works now, the principal fix is not merged yet (https://github.com/lampepfl/dotty/pull/9984), but that's good nonetheless.